scripts/vis_poses.py

```diff
@@ -296,49 +305,197 @@ def main(args):
+                        cbar_title="Error (mm)",
+                        ax=None,
+                        cmap="turbo",
+                        use_horiz_cbar=True,
+                        use_white_bg=True,
+                        include_colorbar=True,
+                        use_fixed_cbar=True,
+                        mask=mask_objs_gt_merged,
+                    )
+                    save_path = vis_path_base(suffix="_depth_heatmap")
+                    plt.savefig(save_path, dpi=100, bbox_inches='tight')
+                if "contour" in args.extra_vis_types:
+                    contour_img = copy.deepcopy(rgb)
+                    for idx in range(len(res_per_obj_est)):
+                        depth_obj_gt = res_per_obj_gt[idx]["depth"]
+                        depth_obj = res_per_obj_est[idx]["depth"]
+                        depth_obj_mask = depth_obj_gt > 0
+                        mask_obj = mask_objs[idx]
+                        mask_obj_gt = mask_objs_gt[idx]
+
+                        percent = calc_mask_visib_percent(mask_obj, depth_obj_mask)
+                        if percent < 20:
+                            print(f"{idx=} {percent=}")
+                            continue
+
+                        contour_img = draw_pose_contour(
+                            contour_img,
+                            rendered_depth=depth_obj_gt,
+                            mask_visib=mask_obj_gt,
+                            contour_color=(0, 255, 0),
+                        )
+                        contour_img = draw_pose_contour(
+                            contour_img,
+                            rendered_depth=depth_obj,
+                            contour_color=(255, 0, 0),
+                            mask_visib=mask_obj,
+                        )
+                    save_path = vis_path_base(suffix="_contour")
+                    inout.save_im(save_path, contour_img)
+                if "bbox3d" in args.extra_vis_types:
+                    bbox_img = copy.deepcopy(rgb)
+                    for idx in range(len(res_per_obj_est)):
+
+                        obj_id = res_per_obj_gt[idx]["pose"]["obj_id"]
+                        pose_gt = res_per_obj_gt[idx]["pose"]
+                        pose_est = res_per_obj_est[idx]["pose"]
+                        model = models[obj_id]
+                        pts = model["pts"]
+                        mesh = trimesh.Trimesh(vertices=pts, faces=model["faces"])
+                        bbox_3d = trimesh.bounds.corners(
+                            mesh.bounding_box_oriented.bounds
+                        )
+                        depth_obj_gt = res_per_obj_gt[idx]["depth"]
+                        depth_obj_mask = depth_obj_gt > 0
+                        mask_obj = mask_objs[idx]
+                        percent = calc_mask_visib_percent(mask_obj, depth_obj_mask)
+                        if percent < 35:
+                            print(f"{idx=} {percent=}")
```

Comment:
Use misc.log. 
What is this 35 percent value and why are we logging this? Could be put as a GLOBAL_VARIABLE or arg and commented